scripts/docker/CC-Devel/bin/install_dbaccess.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+function cleanup() {
+    echo "Cleaning up Odb temporaries."
+    if  [ -n "${ODB_BUILD_DIR}" ]; then
+        rm -rf "${ODB_BUILD_DIR}"
+    fi
+}
+
+trap cleanup EXIT
```

Comment:
What about SIGTERM, SIGINT and SIGKILL?

Reply:
When program control reaches the trap and there was any error, this statement reduces the dead image size, nothing more. In error case we should not care signals, because the image already unusable.